docs/hsi-tests.d/org.fwupd.hsi.Amd.SpiReplayProtection.json

```diff
@@ -14,5 +14,8 @@
     "enabled": "SPI protections enabled"
   },
   "hsi-level": 2,
+  "capabilities": [
+    "cpu-amd"
+  ],
```

Comment:
@superm1 is capabilities the right word? plus, is there any existing way to identify CPUs like amd64 ix64 or something?

Reply:
> @superm1 is capabilities the right word?

I guess a better word might be `matching-hardware`?

> plus, is there any existing way to identify CPUs like amd64 ix64 or something?

What do you mean?  Identify from where?

Reply:
Ohh, i mean an industry standard token, e.g. CPU_AMD64 or something defined in the kernel maybe.

Reply:
read `/proc/cpuinfo` and look at `vendor_id`.

Reply:
so how about `s/cpu-amd/CPUID:AuthenticAMD`?

Reply:
GUIDs is a great idea.